core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -1365,6 +1365,22 @@ def test_isValidAminoAcidChange(self):
         self.assertIn('HGVSp_Short', record.getMessage())
         self.assertEqual(record.line_number, 8)
 
+    def throw_error_if_hgvsp_short_is_too_long(self):
+        """Test that an error is thrown if HGVSp_Short value is too long."""
+        # set level according to this test case:
+        self.logger.setLevel(logging.WARNING)
+        record_list = self.validate(
+                'mutations/data_mutations_too_long_hgvsp_short.maf',
+                validateData.MutationsExtendedValidator,
+                extra_meta_fields={'swissprot_identifier': 'accession'})
+        self.assertEqual(len(record_list), 2)
+        record_iterator = iter(record_list)
+        # expect an error for the second entry
```

Comment:
Should we assert that there is no error logged for the first record? If some logic flaw in the scripts package caused flagging every record as too long I'm sure we would notice, so maybe we don't have to worry about it - but since there are two records in the test set, it would feel more complete to test the code behavior for both cases. I'm not sure how you assert an absence of error .. maybe self.assertNotEqual(record.levelno, logging.ERROR)?

However, I think this PR can be merged without this added test.

Reply:
Good catch! I've fixed the test